Python/initconfig.c

```diff
@@ -260,6 +261,20 @@ static const char usage_envvars[] =
 "                  various kinds of output.  Setting it to 0 deactivates\n"
 "                  this behavior.\n"
 "PYTHON_HISTORY  : the location of a .python_history file.\n"
+"PYTHONASYNCIODEBUG: enable asyncio debug mode\n"
+#ifdef Py_DEBUG
+"PYTHONDUMPREFS  : dump objects and reference counts still alive after shutdown\n"
+"PYTHONDUMPREFSFILE: dump objects and reference counts to the specified file\n"
+#endif
+#ifdef __APPLE__
+"PYTHONEXECUTABLE: set sys.argv[0] to this value (macOS only)\n"
```

Comment:
It's the other way around: *Python* sets `PYTHONEXECUTABLE` on macOS. IMO, it shouldn't be on this list.

Reply:
Thank you for reviewing this! I'll remove it then.

Do you know why the following test passes on macOS? Is it being skipped?

```
    @unittest.skipUnless(sys.platform == 'darwin', 'PYTHONEXECUTABLE only works on macOS')
    def test_python_executable(self):
        code = 'import sys; print(sys.executable)'
        expected = "/busr/bbin/bpython"
        rc, out, err = assert_python_ok('-c', code, PYTHONEXECUTABLE=expected)
        self.assertIn(expected.encode(), out)
```

Reply:
Oh! My bad, I misread the code, and didn't start up a Mac to actually test. Sorry!

I'll revert the latest commit and merge.

Reply:
No problem, thank you again for reviewing, editing and merging!